### ISSUE TITLE
Use John Papa's Style guide

### ISSUE THREAD

**Issue by dirkpostma** [mwaylabs/generator-m-ionic#436]
In angular world, the style guide by John Papa is used more and more. In Angular 2, it's even become part of the official documentation. I would like to suggest to adopt John Papa's style guide in m-generator-ionic.

https://github.com/johnpapa/angular-styleguide

**Comment by gruppjo** [comment]
Hi @dirkpostma. We've been discussing this for a long time. We'll definitely adopt it for version 2, however not for 1. It's historically grown. We started developing the generator before the community was fixed on any particular style and integrating it now would require a lot of time that we see better invested in version 2.
For now.
Not saying that it will never come for version 1 as well, but not soon.
Hope that is a satisfying answer!
